static/scss/dashboard.scss

```diff
@@ -37,4 +37,18 @@ $featureImgMobileWidth: 343px;
       }
     }
   }
-}
\ No newline at end of file
+}
+
+.courses-heading {
+  font-size: 40px;
```

Comment:
This font-size should be applied to the general style in `common.scss` so all `h1`'s use it. There is no reason to have some `h1` elements at 40px and others at 42px/48px. Those values are close enough that you should always assume that the change should be applied to all elements like it. You can also try applying the mobile styles to the `common.scss` and see how the other `h1` elements on other pages look.

You can also remove the `courses-heading` class from that `h1` if the above suggestions work out